nemo/collections/nlp/modules/common/megatron/language_model.py

```diff
@@ -47,10 +48,7 @@
 except (ImportError, ModuleNotFoundError):
 
     HAVE_APEX = False
-
-    # fake missing classes with None attributes
-    AttnMaskType = ApexGuardDefaults()
-    LayerType = ApexGuardDefaults()
+    from nemo.collections.nlp.modules.common.megatron.enums import AttnMaskType, LayerType, ModelType
```

Comment:
What about apex path for the LayerType, ModelType symbols?

Reply:
no apex path will be touch in this pr